lib/member_page.rb

```diff
@@ -6,59 +6,45 @@ class MemberPage < Scraped::HTML
   decorator Scraped::Response::Decorator::CleanUrls
 
   field :id do
-    url[/(\d+)$/, 1]
+    url[/\d+/]
```

Comment:
I'm curious why you changed this one. Is there a scenario now where that _isn't_ at the end? Would be worth adding a test-case for that if so.